fast_llm/models/gpt/conversion.py

```diff
@@ -258,6 +258,7 @@ def _create_config_converters(cls) -> list[ParamConverter]:
             # TODO: Llama supports biases
             ConstantExportParamConverter(None, "attention_bias", False),
             ConstantExportParamConverter(None, "mlp_bias", False),
+            # TODO: Convert rope_scaling (llama3 scaling)
```

Comment:
Are you planning to do in a separate PR?

Reply:
Yes, if that's ok